deps/build.jl

```diff
@@ -34,6 +34,10 @@ if is_apple()
     provides(Homebrew.HB, "homebrew/science/hdf5", hdf5, os=:Darwin)
 end
 
+if Sys.KERNEL === :FreeBSD
```

Comment:
Maybe it's best to just remove the `is_linux` branch and put these all together.